api/controllers/apps_test.go

```diff
@@ -68,14 +76,15 @@ func TestAppCreate(t *testing.T) {
 		err := json.Unmarshal([]byte(body), &resp)
 
 		if assert.Nil(t, err) {
-			assert.Equal(t, "application", resp["name"])
+			assert.Equal(t, "convox-test-application", resp["name"])
```

Comment:
the /apps API should still return the logical app name without the stack prefix.